cypress/e2e/timeline/timeline.spec.ts

```diff
@@ -453,6 +453,118 @@ describe("Timeline", () => {
+
+            // Create a bot "BotBob" and invite it
+            let bot: MatrixClient;
+            cy.getBot(homeserver, {
+                displayName: "BotBob",
+                autoAcceptInvites: false,
+            }).then((_bot) => {
+                bot = _bot;
+                cy.inviteUser(roomId, bot.getUserId());
+                bot.joinRoom(roomId);
+
+                // Make sure the bot joined the room
+                cy.contains(
+                    ".mx_GenericEventListSummary .mx_EventTile_info.mx_EventTile_last",
+                    "BotBob joined the room",
+                ).should("exist");
+
+                // Have bot send MESSAGE to roomId
+                cy.botSendMessage(bot, roomId, MESSAGE);
+
+                // Have bot send stringJa to roomId
+                cy.botSendMessage(bot, roomId, stringJa);
+            });
+
+            // Send strings to the room
+            const strings = [stringAr, stringEl, stringHe, stringHi, stringJa, stringKo, stringTh, stringZh];
+            strings.forEach((strings) => {
+                sendString(strings);
+            });
+
+            // Send some random messages in English
+            cy.sendEvent(roomId, null, "m.room.message" as EventType, {
+                msgtype: "m.text" as MsgType,
+                body:
+                    "Paragraph above lists\n\n" +
+                    "- item 1\n- item 2\n- item 3\n\n" +
+                    "Paragraph between lists\n\n" +
+                    "1. item 1\n2. item 2\n3. item 3\n\n" +
+                    "Paragraph below lists",
+                format: "org.matrix.custom.html",
+                formatted_body:
+                    "<p>Paragraph above lists</p>\n" +
+                    "<ul>\n<li>item 1</li>\n<li>item 2</li>\n<li>item 3</li>\n</ul>\n" +
+                    "<p>Paragraph between lists</p>\n" +
+                    "<ol>\n<li>item 1</li>\n<li>item 2</li>\n<li>item 3</li>\n</ol>\n" +
+                    "<p>Paragraph below lists</p>\n",
+            });
+            cy.sendEvent(roomId, null, "m.room.message" as EventType, {
+                msgtype: "m.emote" as MsgType,
+                body: "sends an emote",
+            });
+            cy.sendEvent(roomId, null, "m.room.message" as EventType, {
+                msgtype: "m.text" as MsgType,
+                body: `${MESSAGE} by Alan`,
+            });
+
+            // Ensure the last message was sent
+            cy.get(".mx_EventTile_last .mx_EventTile_receiptSent").should("be.visible");
+
+            // Ensure queries are searched, found, and highlighted
+            const queries = [
+                // stringAr, // Broken
+                stringEl,
+                // stringHe, // Broken
+                // stringHi, // Broken
+                stringJa,
+                stringKo,
+                stringTh,
+                stringZh,
+                "Paragraph",
+                "emote",
+                "Lorem",
+            ];
```

Comment:
can we reuse `strings` here?

Does searching for `Paragraph`, `emote`, `Lorem`, give any value here beyond showing that search works for the roman alphabet (which hopefully we already know)?

Reply:
No, because `stringAr`, `stringHe`, and `stringHi` need to be sent but commented out here, otherwise the test fails

ok to remove Paragraph, emote, Lorem

Reply:
https://github.com/matrix-org/matrix-react-sdk/pull/10319/commits/e64b3de23f46025e26fd8ea213767569b10c1ec2

Reply:
> No, because `stringAr`, `stringHe`, and `stringHi` need to be sent but commented out here, otherwise the test fails

Sorry, why do they need to be sent if we aren't testing for them?

Reply:
Because if we include tests right now, the test just fails.

I thought creating a test and mentioning there that there are things to be fixed would make sense.

If you think creating an issue for it is enough, l am going to close the PR.

Reply:
I think maybe you misunderstand me. I have left a [suggestion](https://github.com/matrix-org/matrix-react-sdk/pull/10319#discussion_r1130591742) showing what I mean.

Reply:
What I have suggested is that the test should indicate that those strings can successfully be sent to the room and rendered on the search result (around strings which can be found) as well BUT cannot be found when you try to find themselves via search form.

Reply:
> and rendered on the search result (around strings which can be found) as well 

The test does not check this right now, though.